vividus-tests/src/main/resources/story/integration/JSON - Wait for data in HTTP response.story

```diff
@@ -17,6 +17,18 @@ Scenario: Verify step 'When I save JSON element from `$json` by JSON path `$json
 When I save JSON element from `${response}` by JSON path `<jsonPath>` to scenario variable `numberOfBooks`
 Then `${numberOfBooks}` is equal to `<booksNumber>`
 
+Scenario: Verify step "Then JSON element from `$json` by JSON path `$jsonPath` is equal to `$expectedData`$options"
```

Comment:
I guess this scenario was moved to `JSON.story` and this change was introduced due to invalid merge conflict resolution